main.js

```diff
@@ -21,3 +21,20 @@ function turnCar() {
       break;
   }
 }
+
+function startCar() {
+  var position = 0;
+  var car = document.querySelector('img');
+  if (event.keyCode === 32) {
+    setInterval(function () {
+      position += 10;
+      car.style.position = 'absolute';
+      car.style.height = '50px';
+      car.style.width = '10%';
```

Comment:
These pieces of styling should be applied in the stylesheet, since they don;t need to be applied dynamically